### PR TITLE
Ankit | Stage | Prod - Blank page is getting displayed on tax reported in US company when tax number not present in company

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwfyw8u
https://app.clickup.com/t/86cwc8dt8
https://app.clickup.com/t/86cwahbm9
https://app.clickup.com/t/86cwbx294
https://app.clickup.com/t/86cwm3cyb

### CODE REVIEW

File: apps/web-giddh/src/app/shared/header/header.component.scss [hunk 1]
```diff
@@ -106,7 +106,7 @@
 
     ul {
         display: flex;
-        column-gap: 24px;
+        column-gap: 20px;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/header/header.component.scss]: use class "d-flex column-gap2"

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

**Review by dvCodeWorld — DISMISSED** [review]

File: apps/web-giddh/src/app/shared/header/header.component.scss (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/header/header.component.scss]: Done

**Review by dilpreetsinghofficial — DISMISSED** [review]

File: apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.ts [hunk 2]
```diff
@@ -112,13 +114,18 @@ export class TaxSidebarComponent implements OnInit, OnDestroy {
...
                 if (this.vatSupportedCountries.includes(activeCompany.countryV2?.alpha2CountryCode)) {
                     this.showTaxMenus = true;
                 } else if (activeCompany.countryV2?.alpha2CountryCode === 'IN') {
                     this.showGstMenus = true;
                 } else if (this.salesTaxSupportedCountries.includes(activeCompany.countryV2?.alpha2CountryCode)) {
                     this.showSalesTaxMenus = true;
+                } else if (this.trnSupportedCountries.includes(activeCompany.countryV2?.alpha2CountryCode)) {
+                    this.showTaxMenus = true;
+                }
+                if (this.localeData) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.ts:127]: remove duplicate code

File: apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.ts [hunk 3]
```diff
@@ -89,6 +90,22 @@ export class RateWiseReportComponent implements OnInit, OnDestroy {
...
         });
+
+        // Subscribe Tax Number Observable
+        this.componentStore.taxNumber$.pipe(takeUntil(this.destroyed$)).subscribe(response => {
+            if (response?.body?.length) {
+                this.getFormControl('taxNumber').patchValue(response.body[0]);
+            }
+        });
+    }
+/**
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.ts]: format code

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/tax-sidebar/tax-sidebar.component.ts:127]: Done

File: apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/theme/tax-authority/reports/rate-wise-report/rate-wise-report.component.ts]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**